From 0b5b0cabdc4c83fbc556cc1cc8f7a19106a8179a Mon Sep 17 00:00:00 2001
From: Tomasz Unger <tomasz.unger@yahoo.pl>
Date: Mon, 23 Feb 2026 12:53:49 +0100
Subject: [PATCH] staging: rtl8723bs: fix spelling mistakes in include files

Fix spelling mistakes in comments found by codespell:
 - sequcne => sequence (rtw_cmd.h)
 - modifiying => modifying (rtw_mlme.h)

Signed-off-by: Tomasz Unger <tomasz.unger@yahoo.pl>
---
 drivers/staging/rtl8723bs/include/rtw_cmd.h  | 2 +-
 drivers/staging/rtl8723bs/include/rtw_mlme.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/staging/rtl8723bs/include/rtw_cmd.h b/drivers/staging/rtl8723bs/include/rtw_cmd.h
index cb44119ce9a9..c4c3edee809d 100644
--- a/drivers/staging/rtl8723bs/include/rtw_cmd.h
+++ b/drivers/staging/rtl8723bs/include/rtw_cmd.h
@@ -559,7 +559,7 @@ struct RunInThread_param {
 Result:
 0x00: success
 0x01: success, and check Response.
-0x02: cmd ignored due to duplicated sequcne number
+0x02: cmd ignored due to duplicated sequence number
 0x03: cmd dropped due to invalid cmd code
 0x04: reserved.
 
diff --git a/drivers/staging/rtl8723bs/include/rtw_mlme.h b/drivers/staging/rtl8723bs/include/rtw_mlme.h
index 2a128568c6df..7591e28707fa 100644
--- a/drivers/staging/rtl8723bs/include/rtw_mlme.h
+++ b/drivers/staging/rtl8723bs/include/rtw_mlme.h
@@ -76,7 +76,7 @@ Each struct __queue has its own locks, already.
 Other items in mlme_priv are protected by mlme_priv.lock, while items in
 xmit_priv are protected by xmit_priv.lock.
 
-To avoid possible dead lock, any thread trying to modifiying mlme_priv
+To avoid possible dead lock, any thread trying to modifying mlme_priv
 SHALL not lock up more than one locks at a time!
 
 The only exception is that queue functions which take the __queue.lock
-- 
2.53.0

